src/server/interfaces/propertyAddress.ts

```diff
@@ -0,0 +1,8 @@
+export interface PropertyAddress {
```

Comment:
Rename PropertyAddress to AddressProperty
I Think that "Property" is more common name than "Address". There may be many properties with a specific implementation for each. For example NameProperty, AddressProperty, PhoneProperty e.t.c.

Reply:
@Evilart86 Property is physical address of office or some building.

Reply:
okey i understand